### DISCUSSION TITLE
So ,I am developing an project for  web application firewall and need some suggestions

### DISCUSSION THREAD

**Discussion by Abhi00kh** [community/community#73396] [Programming Help]
### Body

As I said i am making an project which focuses on improving **web application firewall through using of deep learning and neural network**and i am not an expert when it comes to field of web development and having bit of issue of how do i integrate my neural network model with and firewall and deploy it  and test its result so if possible can some one refer me some similar project or tell my like module structures ,or anything that can help with my project,**Note** : its my first time using git so please ignore if any mistake 

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by faridnec** [comment]
it might be good to check out existing projects or research papers that have explored a similar intersection of web application firewall and deep learning. GitHub is a goldmine for such projects. Look for keywords like "WAF deep learning" or "neural network firewall." for an example [see](https://www.mdpi.com/1424-8220/23/4/2073)
